src/elements/modalDuration/modalDuration.tsx

```diff
@@ -28,23 +28,23 @@ export const ModalDuration = ({
 
   const daysItems: DurationItem[] = Array.from(
     { length: 31 },
-    (index: number) => ({
+    (_,index: number) => ({
```

Comment:
this fix was done already and it causes you conflicts. Also, again, please set prettier to automatically format please.

Reply:
i resolved the conflict now ..